java/java.lsp.server/build.xml

```diff
@@ -31,12 +31,7 @@
     <target name="build-lsp-server" depends="-set-use-jdk-javac,build,test-init,proxy-setup" description="Prepares NetBeans bits for vscode extensions">
         <ant dir="vscapp" target="build-zip" inheritall="false" inheritrefs="false" />
         <unzip src="vscapp/dist/nb-java-lsp-server.zip" dest="${lsp.build.dir}/../" />
-        <mkdir dir="${lsp.build.dir}/bin" />
```

Comment:
It is, of course, OK to have a better launcher for VS Code, but VS Code is not the only LSP client. So it would be nice if ability to run without the VS Code would be preserved.

Reply:
Ability to launch the app without `node` is important also for debugging. Keeping it, use 
```bash
vscode$ ./nbcode/bin/nbcode
```